frontend/amundsen_application/static/js/features/ColumnList/ColumnType/index.tsx

```diff
@@ -121,12 +122,12 @@ export class ColumnType extends React.Component<
         </OverlayTrigger>
         <Modal
           className="column-type-modal"
-          show={this.state.showModal}
+          show={showModal}
           onHide={this.hideModal}
         >
           <Modal.Header closeButton>
             <Modal.Title>
-              <h5 className="main-title">{MODAL_TITLE}</h5>
+              <div className="main-title">{MODAL_TITLE}</div>
```

Comment:
How come we want a div instead of a heading?

Reply:
This was due to an existing console error I was seeing. The modal title that this is wrapped in is using a h4 tag, and the error was `<h5> cannot appear as a child of <h4>`

Reply:
ahhh, I see. Makes sense!